hilgendorf_support/launch/robot_interface_streaming_hilgendorf.launch

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<!--
+  Hilgendorf control
+-->
+<launch>
+  
+  <!-- robot_ip: IP-address of the robot's socket-messaging server -->
+  <arg name="right_robot_ip"/>
+  <arg name="left_robot_ip"/>
+  <arg name="right_reverse_port" default="50001"/>
+  <arg name="left_reverse_port" default="50002"/>
+  <arg name="min_payload"  default="0.0"/>
+  <arg name="max_payload"  default="10.0"/>
+  <arg name="fake_robotiqs" default="true">
+  
+  <!-- launch drivers for right arm -->
+  <include file="$(find hilgendorf_support)/launch/driver_higlendorf.launch">
+    <arg name="robot_ip" value="$(arg right_robot_ip)"/>
+    <arg name="reverse_port" value="$(arg right_reverse_port)"/>
+    <arg name="min_payload"  value="$(arg min_payload)"/>
+    <arg name="max_payload"  value="$(arg max_payload)"/>
+    <arg name="prefix" value="right_"/>
+  </include>
+
+  <include file="$(find hilgendorf_support)/launch/driver_higlendorf.launch">
+    <arg name="robot_ip" value="$(arg left_robot_ip)"/>
+    <arg name="reverse_port" value="$(arg left_reverse_port)"/>
+    <arg name="min_payload"  value="$(arg min_payload)"/>
+    <arg name="max_payload"  value="$(arg max_payload)"/>
+    <arg name="prefix" value="left_"/>
+  </include>
+
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
+
+   <!-- TF Buffer Server -->
+  <node pkg="tf2_ros" type="buffer_server" name="tf2_buffer_server">
+    <param name="buffer_size" value="120.0"/>
+  </node>
+
+  <!-- If robotiq position data is not being published, ignore them-->
+    <!-- robot model -->
+  <include if="$(arg fake_robotiqs)" file="$(find hilgendorf_support)/launch/load_hilgendorf.launch" />
```

Comment:
One additional note...the `joint_states` topic should be namespaced for each sub-robot.  A `joint_state_publisher` node should then be used to combine the joints into a global `/joint_states` (via the topics list parameter, see [here](http://wiki.ros.org/joint_state_publisher)).